src/components/fileUpload.js

```diff
@@ -71,7 +71,7 @@
     };
 
     const clearFiles = e => {
-      e.preventDefault();
+      if (e && e.preventDefault()) e.preventDefault();
```

Comment:
You probably just want to know if `e.preventDefault` is defined, not to test the result of the function.

Reply:
Fixed that!